test/helper/resource.go

```diff
@@ -242,3 +253,129 @@ func NewJob(namespace string, name string) *batchv1.Job {
 		},
 	}
 }
+
```

Comment:
These methods are used only in UT, should they be placed in the corresponding `pkg/estimator/server` package?

Reply:
I think these code will soon be used in the future test, like #697.